rdkit/Chem/PeriodicTable.py

```diff
@@ -128,11 +127,11 @@
 for line in periodicData.split('\n'):
   splitLine = line.split()
   if len(splitLine) > 1:
-    nameTable[splitLine[1]] = (int(splitLine[0]),float(splitLine[6]),int(splitLine[7]),\
-                               int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    nameTable[splitLine[1]] = (int(splitLine[0]), float(splitLine[6]), int(splitLine[7]),
+                               int(splitLine[5]), float(splitLine[2]), float(splitLine[3]),
                                float(splitLine[4]))
-    numTable[int(splitLine[0])] = (splitLine[1],float(splitLine[6]),int(splitLine[7]),\
-                                   int(splitLine[5]),float(splitLine[2]),float(splitLine[3]),
+    numTable[int(splitLine[0])] = (splitLine[1], float(splitLine[6]), int(splitLine[7]),
```

Comment:
The yapf formatter leaves all lines with the \ character at the end unchanged. In this case, the \ is not necessary, so we get the reformatting. Another case is trailing whitespaces in comments. 

My usual workflow is:
- remove a lot of the pylint messages I get, mainly unused imports. 
- Making changes to the python code
- Shift-Cmd-F to run basic formatting (fixes whitespaces)
- yapf over the file

In bitbucket there is an option to ignore whitespace changes. Just googled that and git has this as a _hidden feature_. Add `?w=1` to the URL and whitespace changes are no longer shown in the diff. https://github.com/rdkit/rdkit/pull/1208/files?w=1